src/interfaces/assistants_web/src/components/Welcome.tsx

```diff
@@ -23,7 +23,7 @@ export const Welcome: React.FC<Props> = ({ show, agentId }) => {
   const { data: tools = [], isLoading: isToolsLoading } = useListTools();
   const { contrastText, bg, contrastFill } = useBrandedColors(agentId);
 
-  const isAgent = agentId !== undefined && !isAgentsLoading && !!agent;
+  const isBaseAgent = agent?.id === '';
```

Comment:
we should create a utility function for this

Reply:
yeah will do